src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -56,9 +55,13 @@ public void resetUnjamTimer() {
     m_timer.reset();
   }
 
```

Comment:
Add javadocs here describing what this method is, and what it returns (i.e. @returns the speed of the revolver in RPM)

Reply:
Resolved in commit cceed37

Reply:
Resolved in commit dd412b9